keras_nlp/models/bert/bert_masked_lm_test.py

```diff
@@ -28,22 +28,23 @@ def setUp(self):
         # Setup model.
         self.vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]", "[MASK]"]
         self.vocab += ["the", "quick", "brown", "fox", "."]
+        self.sequence_length = 5
         self.preprocessor = BertMaskedLMPreprocessor(
             BertTokenizer(vocabulary=self.vocab),
             # Simplify our testing by masking every available token.
             mask_selection_rate=1.0,
             mask_token_rate=1.0,
             random_token_rate=0.0,
             mask_selection_length=5,
-            sequence_length=5,
+            sequence_length=self.sequence_length,
         )
         self.backbone = BertBackbone(
-            vocabulary_size=self.preprocessor.tokenizer.vocabulary_size(),
+            vocabulary_size=len(self.vocab),
             num_layers=2,
             num_heads=2,
             hidden_dim=2,
             intermediate_dim=4,
-            max_sequence_length=self.preprocessor.packer.sequence_length,
+            max_sequence_length=self.sequence_length,
```

Comment:
Same here and elsewhere, let's just do `self.preprocessor.sequence_length`

Reply:
Fixed.